### PR TITLE
:sparkles: Added last modified attribute to concepts written in ES

### PR DESCRIPTION
**Pull request by izzyblues**

### CODE REVIEW

File: resources/handler.go [hunk 1]
```diff
@@ -148,11 +148,18 @@ func (h *Handler) processConceptModel(uuid string, conceptType string, body []by
...
 	}
 
-	payload, err := h.modelPopulator.ConvertConceptToESConceptModel(concept, conceptType)
+	transactionID, err := tid.GetTransactionIDFromContext(ctx)
+
+	if err != nil {
+		transactionID = tid.NewTransactionID()
+		log.WithError(err).WithField(tid.TransactionIDKey, transactionID).Warn("Transaction ID not found to process concept model. Generated new transaction ID")
+	}
+
```
Comment (galiri) [inline resources/handler.go]: should this be tested?

File: service/model.go [hunk 2]
```diff
@@ -33,13 +33,15 @@ type SourceConcept struct {
...
-	DirectType  string   `json:"directType"`
-	Aliases     []string `json:"aliases,omitempty"`
+	Id                string   `json:"id"`
+	ApiUrl            string   `json:"apiUrl"`
+	PrefLabel         string   `json:"prefLabel"`
+	Types             []string `json:"types"`
+	Authorities       []string `json:"authorities"`
+	DirectType        string   `json:"directType"`
+	Aliases           []string `json:"aliases,omitempty"`
+	LastModifiedEpoch int64    `json:"lastModifiedEpoch"`
```
Comment (khatton-ft) [inline service/model.go]: Wouldn't it be nicer to have a human-readable value, like we do in MongoDB?

File: resources/handler.go (hunk 1 above)
Comment (izzyblues) [reply resources/handler.go]: Okay. I am going to write a unit tests for this.

File: service/model.go (hunk 2 above)
Comment (izzyblues) [reply service/model.go]: If you put a string that contains a readable timestamp you need to specify that the field is a date in the mappings to be able to execute a time range query in ES (see [here](https://www.elastic.co/guide/en/elasticsearch/reference/5.1/date.html)).
With the epoch you can execute a numeric range query without any change in the mapping.

**Pull request merged**